modules/assim.batch/R/pda.emulator.R

```diff
@@ -194,13 +321,33 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
       
       ## Convert probabilities back to parameter values
       for(i in 1:n.param) {
-        m[,i] <- eval(prior.fn$qprior[prior.ind][[i]], list(p=mcmc.out[[c]][,i]))
+        m[,i] <- eval(prior.fn$qprior[prior.ind][[i]], list(p=mcmc.out[[c]]$mcmc[,i]))
       }
     }
-      colnames(m) <- pname[prior.ind]
-      mcmc.out[[c]] <- m
+    colnames(m) <- pname[prior.ind]
+    mcmc.list.tmp[[c]] <- m
+    
+    jvar.list[[c]] <- sqrt(mcmc.out[[c]]$jump@history[nrow(mcmc.out[[c]]$jump@history),])
```

Comment:
yes, I'll need to revise the mcmc.GP function later, right now I just have some hacks to make it work